python/astro_metadata_translator/headers.py

```diff
@@ -351,18 +351,19 @@ def _find_from_resource(
         Name of resource read. `None` if no corrections found.
     """
     if package is not None and resource_root is not None:
-        resource_name = posixpath.join(resource_root, target_file)
-        if pkg_resources.resource_exists(package, resource_name):
-            log.debug("Applying header corrections from package resource %s:%s", package, resource_name)
-            with pkg_resources.resource_stream(package, resource_name) as fh:
-                corrections = _read_yaml(fh, f"package resource {package}:{resource_name}")
+        resource_path = resources.files(package).joinpath(resource_root, target_file)  # type: ignore
+        if resource_path.is_file():
+            resource_uri = f"resource://{package}/{posixpath.join(resource_root, target_file)}"
```

Comment:
Slightly worried that at some point we might get into trouble with things that are escaped differently in URIs vs. POSIX paths, but it's not worth doing anything about that now.

Reply:
Right, if I used ResourcePath the escaping should be handled directly and the code would use `ResourcePath.join()`.